backtrace-library/build.gradle

```diff
@@ -11,16 +11,16 @@ androidGitVersion {
 
 android {
 
-    compileSdkVersion 30
+    compileSdkVersion 32
```

Comment:
Could you leave a comment why minAPI is 32 so we’ll remember in the future?

Reply:
minSdkVersion is `21`